### PR TITLE
chore: update featurelist content

### PR DESCRIPTION
**Pull request by sowmyadev-kota**
#### Description of changes:
Change the content where it is verbose , thank you @hibler13 

Staging site https://feature-list--content.d1ywzrxfkb9wgg.amplifyapp.com/

### Instructions

**If this PR should not be merged upon approval for any reason, please submit as a DRAFT**

Which product(s) are affected by this PR (if applicable)?
- [ ] amplify-cli
- [ ] amplify-ui
- [ ] amplify-studio
- [ ] amplify-hosting
- [ ] amplify-libraries

Which platform(s) are affected by this PR (if applicable)?
- [ ] JS
- [ ] iOS
- [ ] Android
- [ ] Flutter
- [ ] React Native

**Please add the product(s)/platform(s) affected to the PR title**

#### Checks

- [ ] Does this PR conform to [the styleguide](https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md)?

- [ ] Does this PR include filetypes other than markdown or images? Please add or update unit tests accordingly.

- [ ] Are any files being deleted with this PR? If so, have the needed redirects been created?

- [ ] Are all links in MDX files using the MDX link syntax rather than HTML link syntax? <br /> 
      _ref: MDX: `[link](https://link.com)` 
            HTML: `<a href="https://link.com">link</a>`_

### When this PR is ready to merge, please check the box below
- [ ] Ready to merge

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._

### CODE REVIEW

File: src/constants/feature-lists-data.ts [hunk 1]
```diff
@@ -98,7 +98,7 @@ const featureListData = {
         items: [
           {
             content:
-              'Add auth to apps using the Amplify CLI and Studio. Support for login, MFA, social providers, and CAPTCHA protection. Integrate auth with intuitive client library APIs.',
+              'Easily configure auth for your app with Amplify CLI and Studio—supports login, MFA, social providers, and more. Then integrate auth with intuitive client library APIs.',
             linkText: 'Simple Configuration',
```
Comment (hibler13) [inline src/constants/feature-lists-data.ts:102]: This should be "Simple configuration"

Note the lowercase 'c'

File: src/constants/feature-lists-data.ts [hunk 2]
```diff
@@ -123,19 +123,19 @@ const featureListData = {
...
             content:
-              'Fetch and mutate data through generated GraphQL queries and mutations from your frontend. Subscriptions allow real-time data sync. Integrate backend data sources like DynamoDB easily.',
+              'Fetch and mutate data through generated GraphQL queries and mutations from your frontend. Sync data in real time and integrate backend data sources easily.',
             linkText: 'Seamless real-time data access',
             link: 'build-a-backend/graphqlapi/subscribe-data/'
           },
           {
             content:
-              'Implement fine-grained authorization rules and policies to securely access data. Control auth at the API level or on individual fields. Manage access with AWS IAM policies or your own auth provider.',
+              'Fetch and mutate data through generated GraphQL queries and mutations from your frontend. Implement fine-grained authorization to securely access data, while controlling auth at the API level or on individual fields. Manage access with AWS IAM policies or your own auth provider.',
```
Comment (hibler13) [inline src/constants/feature-lists-data.ts:138]: Delete the first sentence "Fetch and mutate data..." This doesn't belong in the "Granular authorization" section.

File: src/constants/feature-lists-data.ts [hunk 3]
```diff
@@ -123,19 +123,19 @@ const featureListData = {
...
             linkText: 'Granular Authorization',
```
Comment (hibler13) [inline src/constants/feature-lists-data.ts:139]: Lowercase "authorization"

File: src/constants/feature-lists-data.ts [hunk 4]
```diff
@@ -191,7 +191,7 @@ const featureListData = {
         items: [
           {
             content:
-              'Add auth to apps using the Amplify CLI and Studio. Support for login, MFA, social providers, and CAPTCHA protection. Integrate auth with intuitive client library APIs.',
+              'Easily configure auth for your app with Amplify CLI and Studio—supports login, MFA, social providers, and more. Then integrate auth with intuitive client library APIs.',
             linkText: 'Simple Configuration',
```
Comment (hibler13) [inline src/constants/feature-lists-data.ts:195]: Lowercase "configuration"

File: src/constants/feature-lists-data.ts [hunk 5]
```diff
@@ -216,19 +216,19 @@ const featureListData = {
...
             content:
-              'Fetch and mutate data through generated GraphQL queries and mutations from your frontend. Subscriptions allow real-time data sync. Integrate backend data sources like DynamoDB easily.',
+              'Fetch and mutate data through generated GraphQL queries and mutations from your frontend. Sync data in real time and integrate backend data sources easily.',
             linkText: 'Seamless real-time data access',
             link: 'build-a-backend/graphqlapi/subscribe-data/'
           },
           {
             content:
-              'Implement fine-grained authorization rules and policies to securely access data. Control auth at the API level or on individual fields. Manage access with AWS IAM policies or your own auth provider.',
+              'Fetch and mutate data through generated GraphQL queries and mutations from your frontend. Implement fine-grained authorization to securely access data, while controlling auth at the API level or on individual fields. Manage access with AWS IAM policies or your own auth provider.',
```
Comment (hibler13) [inline src/constants/feature-lists-data.ts:231]: This text is incorrect for "Granular authorization." It should read: Implement fine-grained authorization to securely access data, while controlling auth at the API level or on individual fields. Manage access with AWS IAM policies or your own auth provider.

File: src/constants/feature-lists-data.ts [hunk 6]
```diff
@@ -216,19 +216,19 @@ const featureListData = {
...
             linkText: 'Granular Authorization',
```
Comment (hibler13) [inline src/constants/feature-lists-data.ts:232]: Lowercase "authorization"

File: src/constants/feature-lists-data.ts [hunk 7]
```diff
@@ -284,7 +284,7 @@ const featureListData = {
         items: [
           {
             content:
-              'Add auth to apps using the Amplify CLI and Studio. Support for login, MFA, social providers, and CAPTCHA protection. Integrate auth with intuitive client library APIs.',
+              'Easily configure auth for your app with Amplify CLI and Studio—supports login, MFA, social providers, and more. Then integrate auth with intuitive client library APIs.',
             linkText: 'Simple Configuration',
```
Comment (hibler13) [inline src/constants/feature-lists-data.ts:288]: Lowercase "configuration"

File: src/constants/feature-lists-data.ts [hunk 8]
```diff
@@ -309,19 +309,19 @@ const featureListData = {
...
             content:
-              'Fetch and mutate data through generated GraphQL queries and mutations from your frontend. Subscriptions allow real-time data sync. Integrate backend data sources like DynamoDB easily.',
+              'Fetch and mutate data through generated GraphQL queries and mutations from your frontend. Sync data in real time and integrate backend data sources easily.',
             linkText: 'Seamless real-time data access',
             link: 'build-a-backend/graphqlapi/subscribe-data/'
           },
           {
             content:
-              'Implement fine-grained authorization rules and policies to securely access data. Control auth at the API level or on individual fields. Manage access with AWS IAM policies or your own auth provider.',
+              'Fetch and mutate data through generated GraphQL queries and mutations from your frontend. Implement fine-grained authorization to securely access data, while controlling auth at the API level or on individual fields. Manage access with AWS IAM policies or your own auth provider.',
```
Comment (hibler13) [inline src/constants/feature-lists-data.ts:324]: This is not the right text for "Granular authorization." Should be: Implement fine-grained authorization to securely access data, while controlling auth at the API level or on individual fields. Manage access with AWS IAM policies or your own auth provider.

File: src/constants/feature-lists-data.ts [hunk 9]
```diff
@@ -309,19 +309,19 @@ const featureListData = {
...
             linkText: 'Granular Authorization',
```
Comment (hibler13) [inline src/constants/feature-lists-data.ts:325]: Lowercase "authorization"

File: src/constants/feature-lists-data.ts [hunk 10]
```diff
@@ -384,7 +384,7 @@ const featureListData = {
         items: [
           {
             content:
-              'Add auth to apps using the Amplify CLI and Studio. Support for login, MFA, social providers, and CAPTCHA protection. Integrate auth with intuitive client library APIs.',
+              'Easily configure auth for your app with Amplify CLI and Studio—supports login, MFA, social providers, and more. Then integrate auth with intuitive client library APIs.',
             linkText: 'Simple Configuration',
```
Comment (hibler13) [inline src/constants/feature-lists-data.ts:388]: Lowercase "configuration"

File: src/constants/feature-lists-data.ts [hunk 11]
```diff
@@ -409,19 +409,19 @@ const featureListData = {
...
             content:
-              'Fetch and mutate data through generated GraphQL queries and mutations from your frontend. Subscriptions allow real-time data sync. Integrate backend data sources like DynamoDB easily.',
+              'Fetch and mutate data through generated GraphQL queries and mutations from your frontend. Sync data in real time and integrate backend data sources easily.',
             linkText: 'Seamless real-time data access',
             link: 'build-a-backend/graphqlapi/subscribe-data/'
           },
           {
             content:
-              'Implement fine-grained authorization rules and policies to securely access data. Control auth at the API level or on individual fields. Manage access with AWS IAM policies or your own auth provider.',
+              'Fetch and mutate data through generated GraphQL queries and mutations from your frontend. Implement fine-grained authorization to securely access data, while controlling auth at the API level or on individual fields. Manage access with AWS IAM policies or your own auth provider.',
```
Comment (hibler13) [inline src/constants/feature-lists-data.ts:424]: Incorrect text for "Granular authorization" section. Should be: Implement fine-grained authorization to securely access data, while controlling auth at the API level or on individual fields. Manage access with AWS IAM policies or your own auth provider.

File: src/constants/feature-lists-data.ts [hunk 12]
```diff
@@ -409,19 +409,19 @@ const featureListData = {
...
             linkText: 'Granular Authorization',
```
Comment (hibler13) [inline src/constants/feature-lists-data.ts:425]: Lowercase "authorization"

File: src/constants/feature-lists-data.ts [hunk 13]
```diff
@@ -477,7 +477,7 @@ const featureListData = {
         items: [
           {
             content:
-              'Add auth to apps using the Amplify CLI and Studio. Support for login, MFA, social providers, and CAPTCHA protection. Integrate auth with intuitive client library APIs.',
+              'Easily configure auth for your app with Amplify CLI and Studio—supports login, MFA, social providers, and more. Then integrate auth with intuitive client library APIs.',
             linkText: 'Simple Configuration',
```
Comment (hibler13) [inline src/constants/feature-lists-data.ts:481]: Lowercase "configuration"

File: src/constants/feature-lists-data.ts [hunk 14]
```diff
@@ -502,19 +502,19 @@ const featureListData = {
...
             content:
-              'Fetch and mutate data through generated GraphQL queries and mutations from your frontend. Subscriptions allow real-time data sync. Integrate backend data sources like DynamoDB easily.',
+              'Fetch and mutate data through generated GraphQL queries and mutations from your frontend. Sync data in real time and integrate backend data sources easily.',
             linkText: 'Seamless real-time data access',
             link: 'build-a-backend/graphqlapi/subscribe-data/'
           },
           {
             content:
-              'Implement fine-grained authorization rules and policies to securely access data. Control auth at the API level or on individual fields. Manage access with AWS IAM policies or your own auth provider.',
+              'Fetch and mutate data through generated GraphQL queries and mutations from your frontend. Implement fine-grained authorization to securely access data, while controlling auth at the API level or on individual fields. Manage access with AWS IAM policies or your own auth provider.',
```
Comment (hibler13) [inline src/constants/feature-lists-data.ts:517]: Incorrect text. Update as detailed above.

File: src/constants/feature-lists-data.ts [hunk 15]
```diff
@@ -502,19 +502,19 @@ const featureListData = {
...
             linkText: 'Granular Authorization',
```
Comment (hibler13) [inline src/constants/feature-lists-data.ts:518]: Lowercase "authorization"

File: src/constants/feature-lists-data.ts [hunk 16]
```diff
@@ -576,7 +576,7 @@ const featureListData = {
         items: [
           {
             content:
-              'Add auth to apps using the Amplify CLI and Studio. Support for login, MFA, social providers, and CAPTCHA protection. Integrate auth with intuitive client library APIs.',
+              'Easily configure auth for your app with Amplify CLI and Studio—supports login, MFA, social providers, and more. Then integrate auth with intuitive client library APIs.',
             linkText: 'Simple Configuration',
```
Comment (hibler13) [inline src/constants/feature-lists-data.ts:580]: Lowercase "configuration"

File: src/constants/feature-lists-data.ts [hunk 17]
```diff
@@ -601,19 +601,19 @@ const featureListData = {
...
             content:
-              'Fetch and mutate data through generated GraphQL queries and mutations from your frontend. Subscriptions allow real-time data sync. Integrate backend data sources like DynamoDB easily.',
+              'Fetch and mutate data through generated GraphQL queries and mutations from your frontend. Sync data in real time and integrate backend data sources easily.',
             linkText: 'Seamless real-time data access',
             link: 'build-a-backend/graphqlapi/subscribe-data/'
           },
           {
             content:
-              'Implement fine-grained authorization rules and policies to securely access data. Control auth at the API level or on individual fields. Manage access with AWS IAM policies or your own auth provider.',
+              'Fetch and mutate data through generated GraphQL queries and mutations from your frontend. Implement fine-grained authorization to securely access data, while controlling auth at the API level or on individual fields. Manage access with AWS IAM policies or your own auth provider.',
```
Comment (hibler13) [inline src/constants/feature-lists-data.ts:616]: Update text as described above and lowercase "authorization" in the LinkText.

File: src/constants/feature-lists-data.ts [hunk 18]
```diff
@@ -676,7 +676,7 @@ const featureListData = {
         items: [
           {
             content:
-              'Add auth to apps using the Amplify CLI and Studio. Support for login, MFA, social providers, and CAPTCHA protection. Integrate auth with intuitive client library APIs.',
+              'Easily configure auth for your app with Amplify CLI and Studio—supports login, MFA, social providers, and more. Then integrate auth with intuitive client library APIs.',
             linkText: 'Simple Configuration',
```
Comment (hibler13) [inline src/constants/feature-lists-data.ts:680]: Lowercase "configuration"

File: src/constants/feature-lists-data.ts [hunk 19]
```diff
@@ -701,19 +701,19 @@ const featureListData = {
...
             content:
-              'Fetch and mutate data through generated GraphQL queries and mutations from your frontend. Subscriptions allow real-time data sync. Integrate backend data sources like DynamoDB easily.',
+              'Fetch and mutate data through generated GraphQL queries and mutations from your frontend. Sync data in real time and integrate backend data sources easily.',
             linkText: 'Seamless real-time data access',
             link: 'build-a-backend/graphqlapi/subscribe-data/'
           },
           {
             content:
-              'Implement fine-grained authorization rules and policies to securely access data. Control auth at the API level or on individual fields. Manage access with AWS IAM policies or your own auth provider.',
+              'Fetch and mutate data through generated GraphQL queries and mutations from your frontend. Implement fine-grained authorization to securely access data, while controlling auth at the API level or on individual fields. Manage access with AWS IAM policies or your own auth provider.',
```
Comment (hibler13) [inline src/constants/feature-lists-data.ts:716]: Update as described above and lowercase "authorization" in LinkText.

File: src/constants/feature-lists-data.ts [hunk 20]
```diff
@@ -775,7 +775,7 @@ const featureListData = {
         items: [
           {
             content:
-              'Add auth to apps using the Amplify CLI and Studio. Support for login, MFA, social providers, and CAPTCHA protection. Integrate auth with intuitive client library APIs.',
+              'Easily configure auth for your app with Amplify CLI and Studio—supports login, MFA, social providers, and more. Then integrate auth with intuitive client library APIs.',
             linkText: 'Simple Configuration',
```
Comment (hibler13) [inline src/constants/feature-lists-data.ts:779]: Lowercase "configuration"

File: src/constants/feature-lists-data.ts [hunk 21]
```diff
@@ -800,19 +800,19 @@ const featureListData = {
...
             content:
-              'Fetch and mutate data through generated GraphQL queries and mutations from your frontend. Subscriptions allow real-time data sync. Integrate backend data sources like DynamoDB easily.',
+              'Fetch and mutate data through generated GraphQL queries and mutations from your frontend. Sync data in real time and integrate backend data sources easily.',
             linkText: 'Seamless real-time data access',
             link: 'build-a-backend/graphqlapi/subscribe-data/'
           },
           {
             content:
-              'Implement fine-grained authorization rules and policies to securely access data. Control auth at the API level or on individual fields. Manage access with AWS IAM policies or your own auth provider.',
+              'Fetch and mutate data through generated GraphQL queries and mutations from your frontend. Implement fine-grained authorization to securely access data, while controlling auth at the API level or on individual fields. Manage access with AWS IAM policies or your own auth provider.',
```
Comment (hibler13) [inline src/constants/feature-lists-data.ts:815]: Update text as described above and lowercase "authorization" in the LinkText

**Review by hibler13 — CHANGES_REQUESTED** [review]
@sowmyadev-kota This is ready for your review. Please make the requested changes and share it again for review when you're done. Thanks!

File: src/constants/feature-lists-data.ts (hunk 1 above)
Comment (sowmyadev-kota) [reply src/constants/feature-lists-data.ts:102]: Done

File: src/constants/feature-lists-data.ts (hunk 2 above)
Comment (sowmyadev-kota) [reply src/constants/feature-lists-data.ts:138]: Done

File: src/constants/feature-lists-data.ts (hunk 3 above)
Comment (sowmyadev-kota) [reply src/constants/feature-lists-data.ts:139]: Done

File: src/constants/feature-lists-data.ts (hunk 4 above)
Comment (sowmyadev-kota) [reply src/constants/feature-lists-data.ts:195]: Done

File: src/constants/feature-lists-data.ts (hunk 2 above)
Comment (sowmyadev-kota) [reply src/constants/feature-lists-data.ts:138]: Done

File: src/constants/feature-lists-data.ts (hunk 3 above)
Comment (sowmyadev-kota) [reply src/constants/feature-lists-data.ts:139]: Done

File: src/constants/feature-lists-data.ts (hunk 4 above)
Comment (sowmyadev-kota) [reply src/constants/feature-lists-data.ts:195]: Done

File: src/constants/feature-lists-data.ts (hunk 17 above)
Comment (sowmyadev-kota) [reply src/constants/feature-lists-data.ts:616]: Done

File: src/constants/feature-lists-data.ts (hunk 18 above)
Comment (sowmyadev-kota) [reply src/constants/feature-lists-data.ts:680]: Done

File: src/constants/feature-lists-data.ts (hunk 19 above)
Comment (sowmyadev-kota) [reply src/constants/feature-lists-data.ts:716]: Done

File: src/constants/feature-lists-data.ts (hunk 20 above)
Comment (sowmyadev-kota) [reply src/constants/feature-lists-data.ts:779]: Done

File: src/constants/feature-lists-data.ts (hunk 21 above)
Comment (sowmyadev-kota) [reply src/constants/feature-lists-data.ts:815]: Done

**Review by hibler13 — APPROVED** [review]
LGTM

**Review by hibler13 — APPROVED** [review]
LGTM

**Review by reesscot — APPROVED** [review]

**Pull request merged**